templates/default/components/Layout.js

```diff
@@ -31,6 +31,11 @@ export default function Layout({ children, title='' }) {
                 </a>
               </Link>
             ))}
+            <Link href="/_all">
```

Comment:
This shouldn't be hard-coded into the front page. If someone wants a link they can put it in there navlinks themselves i think.

I know this was in acceptance so my responsibility - just corrected acceptance to be clearer.